kiwi/bootloader/config/grub2.py

```diff
@@ -271,12 +270,14 @@ def setup_disk_boot_images(self, boot_uuid, lookup_path=None):
             by the distribution. The bios core image is created
             when grub2-install is called
         """
-        log.info('Creating grub bootloader images')
+        self.__copy_theme_data_to_boot_directory(lookup_path)
 
-        if self.firmware.efi_mode():
-            self.efi_boot_path = self.create_efi_path()
+        if not self.firmware.efi_mode():
+            return
 
-        self.__copy_theme_data_to_boot_directory(lookup_path)
+        log.info('Creating grub bootloader images')
+
+        self.efi_boot_path = self.create_efi_path()
 
```

Comment:
sorry I don't understand this change at all

* why did you move the theme copy some lines above, that has no impact and is not related to ppc support ?
* why did you introduce another condition which returns from the function with no effect ? the lines after this return statement are not evaluated if the firmware mode is not efi and it imho is not related to ppc support ?